tests/Mock/DefaultAggregateRoot.php

```diff
@@ -71,9 +71,9 @@ public function popRecordedEvents()
     }
 
     /**
-     * @param Message $event
+     * @param $event
      */
-    public function apply(Message $event)
+    public function apply($event)
```

Comment:
Was clarified in the chat. But I'll add a short note for history:
We rely on Message interface on the event store side but the mock is used to simulate an aggregate of domain which must not rely on the Message interface as an AggregateTranslator can translate custom domain events into Message compatible events